core/templates/cowdata.h

```diff
@@ -231,9 +231,11 @@ uint32_t CowData<T>::_copy_on_write() {
 		uint32_t current_size = *_get_size();
 
 		uint32_t *mem_new = (uint32_t *)Memory::alloc_static(_get_alloc_size(current_size), true);
+		uint32_t *refcount_offset = mem_new - 2;
+		uint32_t *size_offset = mem_new - 1;
```

Comment:
Should these be const references to allow some optimizations?

`uint32_t * const &refcount_offset = mem_new - 2;`

Reply:
I see no reason why this can't be the case.